bytecode/concrete.py

```diff
@@ -292,7 +291,7 @@ def to_code(self):
                               self.kwonlyargcount,
                               nlocals,
                               stacksize,
-                              self.flags,
+                              self.flags.to_int(self),
```

Comment:
The point here was to pass the bytecode to infer some flag values. If we want to rely on standard methods we need to set the bytecode as an attribute of the flag object.